webapp/src/test/java/com/box/l10n/mojito/service/thirdparty/ThirdPartyTMSSmartlingTest.java

```diff
@@ -179,6 +179,16 @@ public void setUp() {
     Mockito.when(smartlingClient.getRetryConfiguration()).thenReturn(retryConfiguration);
   }
 
+  @Test
+  public void testRemoveImage() {
+    String projectId = "projectId";
+    String contextId = "contextId";
+
+    smartlingClient.deleteContext(projectId, contextId);
```

Comment:
what was the intent here? Unless I'm missing a detail, this is not testing anything.

Reply:
line 189 is testing if the call tp the underlying `smartlingClient. deleteContext()` was correctly made.